initialTranscript.json

```diff
@@ -61760,8 +61763,15 @@
             },
             "witness": {
                 "runningProducts": [ "0x97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb"],
-                "potPubkeys": ["0x93e02b6052719f607dacd3a088274f65596bd0d09920b61ab5da61bbdc7f5049334cf11213945d57e5ac7d055d042b7e024aa2b2f08f0a91260805272dc51051c6e47ad4fa403b02b4510b647ae3d1770bac0326a805bbefd48056c8c121bdb8"]
+                "potPubkeys": ["0x93e02b6052719f607dacd3a088274f65596bd0d09920b61ab5da61bbdc7f5049334cf11213945d57e5ac7d055d042b7e024aa2b2f08f0a91260805272dc51051c6e47ad4fa403b02b4510b647ae3d1770bac0326a805bbefd48056c8c121bdb8"],
+                "bls_signatures": [""]
```

Comment:
same as above bls_signatures -> blsSignatures

Reply:
Fixed in https://github.com/ethereum/kzg-ceremony-specs/pull/33/commits/22680924fe194e85ed00919744d13c90401c76f2